dockertest/subtestbase.py

```diff
@@ -139,6 +139,24 @@ def failif_ne(actual, expected, reason=None):
         spec = "{}: expected " + arg + "; got " + arg
         raise DockerTestFail(spec.format(reason, expected, actual))
 
+    @staticmethod
+    def failif_not_in(needle, haystack, description=None):
+        """
+        Convenience method for subtests to test for an expected substring
+        being contained in a larger string, e.g. to look for XYZ in a
+        command's stdout/stderr.
+
+        :param needle: the string you're looking for
+        :param haystack: the actual string, e.g stdout results from a command
+        :param description: description of haystack, e.g. 'stdout from foo'
```

Comment:
Thanks for the link, but why did I not get it in the github email? Did you edit/revise your comment? Anyway, the link answers my initial question re: whether it's `:raise` or `:raises` (singular vs plural); both are in the code base. Answer: both are valid. 

TODO in a separate checkin: fix `dockertest/containers.py` which has a `:rais` (no 'e'), and several files with `:raise: ExceptionName` (misplaced colon).